requirements_test.txt

```diff
@@ -10,3 +10,4 @@ pytest-profiling~=1.7
 pytest-xdist~=2.5
 # Type packages for mypy
 types-pkg_resources==0.1.3
+numpy~=1.22.3
```

Comment:
I don't think we should add numpy to our standard test requirements, if we do that then we could have code that misbehave if numpy is not installed and we would not know. What we could do is create a CI job where we install additional libraries. It's been a recurring topic when dealing with pandas/numpy, etc. To be clear we need to be able to handle crashes in external lib and make sure that we do not regress but for false positives it's a little blurrier. For django and pytest we deferred to external plugins for false positives. Starting to do lib specific code directly in pylint is a slipery slope, we already have a lot on our plates as it is.

Reply:
I bet I could just rewrite with a `fake_package/distutils`.

Reply:
Nah, that didn't work.

Reply:
I added a comment. Adding a new workflow is not a very appealing ROI right now. We tested this with `numpy` installed in `astroid`, and if it fails somehow here in `pylint` again, a contributor will notice.